plugins/modules/ome_alert_policies_actions_info.py

```diff
@@ -0,0 +1,291 @@
+          "Name": "localhost.scomdev.com:555",
+          "Value": "true",
+          "Type": "boolean",
+          "TemplateParameterTypeDetails": []
+        },
+        {
+          "Id": 2,
+          "Name": "localhost.scomdev.com:555",
+          "Value": "true",
+          "Type": "boolean",
+          "TemplateParameterTypeDetails": []
+        }
+      ]
+    },
+    {
+      "Id": 100,
+      "Name": "Ignore",
+      "Description": "Ignore",
+      "Disabled": false,
+      "ParameterDetails": []
+    },
+    {
+      "Id": 70,
+      "Name": "SMS",
+      "Description": "SMS",
+      "Disabled": false,
+      "ParameterDetails": [
+        {
+          "Id": 1,
+          "Name": "to",
+          "Value": "",
+          "Type": "string",
+          "TemplateParameterTypeDetails": [
+            {
+              "Name": "maxLength",
+              "Value": "255"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "Id": 110,
+      "Name": "PowerControl",
+      "Description": "Power Control Action Template",
+      "Disabled": false,
+      "ParameterDetails": [
+        {
+          "Id": 1,
+          "Name": "powercontrolaction",
+          "Value": "poweroff",
+          "Type": "singleSelect",
+          "TemplateParameterTypeDetails": [
+            {
+              "Name": "option",
+              "Value": "powercycle"
+            },
+            {
+              "Name": "option",
+              "Value": "poweroff"
+            },
+            {
+              "Name": "option",
+              "Value": "poweron"
+            },
+            {
+              "Name": "option",
+              "Value": "gracefulshutdown"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "Id": 111,
+      "Name": "RemoteCommand",
+      "Description": "RemoteCommand",
+      "Disabled": true,
+      "ParameterDetails": [
+        {
+          "Id": 1,
+          "Name": "remotecommandaction",
+          "Value": null,
+          "Type": "singleSelect",
+          "TemplateParameterTypeDetails": []
+        }
+      ]
+    },
+    {
+      "Id": 112,
+      "Name": "Mobile",
+      "Description": "Mobile",
+      "Disabled": false,
+      "ParameterDetails": []
+    }
+  ]
+}
+  ]
+error_info:
+  description: Details of the HTTP Error.
+  returned: on HTTP error
+  type: dict
+  sample: {
+    "error": {
+      "code": "Base.1.0.GeneralError",
+      "message": "A general error has occurred. See ExtendedInfo for more information.",
+      "@Message.ExtendedInfo": [
+        {
+          "MessageId": "GEN1234",
+          "RelatedProperties": [],
+          "Message": "Unable to process the request because an error occurred.",
+          "MessageArgs": [],
+          "Severity": "Critical",
+          "Resolution": "Retry the operation. If the issue persists, contact your system administrator."
+        }
+      ]
+    }
+  }
+msg:
+  description: Successfully retrieved alert policies actions information.
```

Comment:
Description is not correct for msg, Please update.

Reply:
The description was verified by info dev team.